documentation/tutorial.md

```diff
@@ -68,3 +71,622 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+
+App                        Version  Status   Scale  Charm                      Channel   Rev  Exposed  Message
+mongodb                             active       2  mongodb                    dpe/edge   96  no       Replica set primary
+tls-certificates-operator           blocked      1  tls-certificates-operator  edge       16  no       Configuration options missing: ['certificate', 'ca-certificate']
+
+Unit                          Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*                    active    idle   0        10.23.62.156    27017/tcp  Replica set primary
+mongodb/1                     active    idle   1        10.23.62.55     27017/tcp  Replica set secondary
+tls-certificates-operator/0*  blocked   idle   3        10.23.62.8                 Configuration options missing: ['certificate', 'ca-certificate']
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+1        started  10.23.62.55   juju-d35d30-1  focal       Running
+3        started  10.23.62.8    juju-d35d30-3  focal       Running
+```
+
+Now we can configure the TLS certificates. Configure the  `tls-certificates-operator` to use self signed certificates:
+```
+juju config tls-certificates-operator generate-self-signed-certificates="true" ca-common-name="Tutorial CA" 
+```
+*Note: this tutorial uses (self signed certificates)[https://en.wikipedia.org/wiki/Self-signed_certificate]; self signed certificates should not be used in a production cluster.*
+
+### Enabling TLS:
+After configuring the certificates `watch -c juju status --color` will show the status of `tls-certificates-operator` as active. To enable TLS on Charmed MongoDB, relate the two applications:
+```
+juju relate tls-certificates-operator mongodb
+```
+
+### Connecting to MongoDB with TLS:
+Once TLS has been enabled we will need to change how we connect to MongoDB. Specifically we will need to specify the TLS CA file along with the TLS Certificate file. These are hosted on the Charmed MongoDB application. Copy them from Charmed MongoDB to your machine:
+```
+juju run --unit mongodb/0 -- cat /etc/mongodb/external-ca.crt > external-ca.crt
+juju run --unit mongodb/0 -- cat /etc/mongodb/external-cert.pem > external-cert.pem
+```
+
+As before, we will connect to MongoDB via a MongoDB URI; use the admin user MongoDB URI that you first used to [connect to MongoDB](#connect-via-mongodb-uri). Connect using the URI and the following TLS options:
+```
+mongosh mongodb://<username>:<password>@<host1>/<database name>?replicaSet=<replica set name> --tls --tlsCAFile external-ca.crt --tlsCertificateKeyFile external-cert.pem
+```
+
+Congratulations, you've now connected to MongoDB with TLS. Now exit the shell by typing `exit`.
+
+### Disabling TLS
+To disable TLS unrelate the two applications:
+```
+juju remove-relation mongodb tls-certificates-operator
+```
```

Comment:
It is nice to end a tutorial with a summary, and almost certainly with some "Calls to action", or  a list of places to go next. for example, see [microk8s](https://microk8s.io/docs/getting-started#heading--next-steps)

Reply:
Yes! Totally agree. This is our only documentation at the moment so we won't be able to link to something that should be done after or places to go next. I can brain storm something though :)